src/common/ui-components/Video/Video.module.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../../styles/partials/variables';
 
 $video_container_height: 100vh;
-$video_player_offset: -10px;
```

Comment:
> $video_player_offset

Very hacky. The video itself should be fixed to not include black bars.

Nit: now there's a mix of snake case and kebab case in the style variables. Makes the code look quite sloppy.

Reply:
I think we should disable the video altogether before we get a better cropped version (which also doesn't play as fast).

Reply:
well, this is something Aleksanteri asked me to do today for the demo tomorrow. Otherwise I would not do it